apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js

```diff
@@ -138,7 +138,7 @@ describe('TeacherHomepage', () => {
    */
   it('renders CensusTeacherBanner if showCensusBanner is true and forceHide is false', () => {
     const wrapper = setUp({showCensusBanner: true});
-    assert(!wrapper.find('CensusTeacherBanner').exists());
+    assert(wrapper.find('CensusTeacherBanner').exists());
```

Comment:
I'm curious what's happening here and on line 150 below?

Reply:
Were removing the AFE banner (which the assert on line 150 is looking for) and showing the CensusTeacherBanner, which this line is looking for.